tools/ci/setupCompiler.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
why not put these to tools/github/... so its clear where its used?

Reply:
Because I wanted to avoid having to move them around and have the possibility to extend to other cis when required. E.g. The build.sh can already be used by github, Jenkins and travis 
These 2 might be as well reusable but there is no need yet